example/image-classification/train_mnist.py

```diff
@@ -75,5 +75,13 @@ def get_mnist_iter(args, kv):
     net = import_module('symbols.'+args.network)
     sym = net.get_symbol(**vars(args))
 
+    # tensorboard logs
+    train_log = 'logs/mnist/train'
+    eval_log = 'logs/mnist/eval'
+    batch_end_callbacks = [mx.contrib.tensorboard.LogMetricsCallback(train_log)]
```

Comment:
If some one hasn't install tensorboard yet, it might throw an exception. Consider making it an optional feature.